packages/eslint-plugin/src/rules/dependency-checks.ts

```diff
@@ -45,7 +45,7 @@ export default createESLintRule<Options, MessageIds>({
         type: 'object',
         properties: {
           buildTargets: [{ type: 'string' }],
-          ignoreDependencies: [{ type: 'string' }],
+          ignoredDependencies: [{ type: 'string' }],
```

Comment:
Optionally, instead of adding another param, we can expose peer dependencies as a nested structure under parent dependency, so that source function (e.g. linter rule) can easily discard all peer dependencies of the ignored packages.